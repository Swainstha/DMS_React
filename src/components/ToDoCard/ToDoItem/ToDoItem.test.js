import testSnapFunction from '../../../util/testSnapFunction';
import ToDoItem from './index'
import React from 'react';
const dummy_data={
todo:{
    selected:true,
    notes:"notes"
},
onToDoChecked:jest.fn()
}
testSnapFunction("<ToDoItem/>", "Snapshot test for ToDoItem",<ToDoItem {...dummy_data}/> )
