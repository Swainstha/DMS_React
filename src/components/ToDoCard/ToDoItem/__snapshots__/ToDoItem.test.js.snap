// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToDoItem/> Snapshot test for ToDoItem 1`] = `ShallowWrapper {}`;
